deface/deface.py

```diff
@@ -18,6 +19,31 @@
 from deface.centerface import CenterFace
 
 
+class ThresholdTimeline:
```

Comment:
Thanks for the class! I think it makes the code way more readable.
Since it's non-trivial code, it's maybe worth adding a couple of tests to document it?